docs/src/advanced/symmetries.md

```diff
@@ -3,41 +3,41 @@
 In this discussion we will only describe the situation for a monoatomic crystal
 ``\mathcal C \subset \mathbb R^3``, the extension being easy.
 A symmetry of the crystal is a real-space unitary matrix ``\tilde{S}``
-and a real-space vector ``\tilde{τ}`` such that
+and a real-space vector ``\tilde{\tau}`` such that
 ```math
 \tilde{S} \mathcal{C} + \tilde{\tau} = \mathcal{C}.
 ```
-The symmetries where ``\tilde S = 1`` and ``\tilde{τ}``
+The symmetries where ``\tilde S = 1`` and ``\tilde{\tau}``
 is a lattice vector are always assumed and ignored in the following.
 
-We can define a corresponding unitary operator
-``{U} : L^2_\text{per} \to L^2_\text{per}``
+We can define a corresponding unitary operator ``U`` on ``L^2(\mathbb R^3)``
 with action
 ```math
- (Uu)(x) = u\left( S^{-1} (x-\tau) \right),
+ (Uu)(x) = u\left( \tilde S x + \tilde \tau \right).
```

Comment:
curly braces around the thing tilde acts on

Reply:
that actually doesn't matter. However we should do \widetilde everywhere

Reply:
Please put the braces in for stuff that is permanent. You too easily introduce an error by writing something before.